README.md

```diff
@@ -564,6 +571,8 @@ _all_ of [`FieldState`](https://github.com/final-form/final-form#fieldstate).
 A function that takes the field value, all the values of the form and the `meta` data about the field and
 returns an error if the value is invalid, or `undefined` if the value is valid.
 
+⚠️ IMPORTANT ⚠️ – By default, in order to allow inline fat-arrow validation functions, the field will not rerender if you change your validation function to an alternate function that has a different behavior. If you need your field to rerender with a new validation function, you will need to update another prop on the `Field`, such as `key`.
```

Comment:
My comment [here](https://github.com/final-form/react-final-form/pull/467#discussion_r283669309) was more about this IMPORTANT note rather the other code line. This note here is kinda confusing to me - is `validate` updatable? As in - will the "latest" validate be used or just the first one? 

If it's updatable when knowing that it doesn't cause a rerender would matter? Also what wouldnt rerender in this case?

Reply:
Maybe this will clear it up? If you don't include [this line here](https://github.com/final-form/react-final-form/blob/hooks/src/Field.test.js#L455), the field won't rerender again as invalid after swapping out the validation function.

It _is_ updatable, but if you need a rerender, you've gotta change a prop that will cause it. Make more sense? Is there a better way to word the IMPORTANT note?

Reply:
Hm, ok - I get it. Not sure though if this has to be said - it sounds complex, but in reality you shouldnt use `validate` inside of your function-child, so you indeed doesnt have to rerender when it changes. OTOH it's obviously good to mention/document every edge case. 🤷‍♂ 

Maybe something like:
> You shouldn't rely on `validate` inside of your component, just let the Final Form handle that for you. If you really need to rerender `Field` along with `validate` change then force the rerender by a new value of some other prop.